src/pages/home/report/ReportActionItem.js

```diff
@@ -90,6 +117,23 @@ class ReportActionItem extends Component {
         this.textInput.focus();
     }
 
+    /**
+     * Animate the background of the item
+     */
+    animateBackground() {
+        Animated.timing(this.animatedBackgroundColor, {
+            toValue: 1,
+            duration: 0,
+            useNativeDriver: false,
+        }).start(() => {
+            Animated.timing(this.animatedBackgroundColor, {
+                toValue: 0,
+                duration: 5500,
```

Comment:
NAB, can't animation values be stored in constants so they can be reused for consistency?

Reply:
No i personally dont see any gain in storing these in constants since its super specific to the animation. Maybe someday if we have a doc centralizing animation timers we can do this but for now i think its fine as is.